tests/custom-elements/reactions/HTMLButtonElement.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
head, html, body labels are unnecessary.

Reply:
Labels head and html have been omitted.